packages/core/src/index.ts

```diff
@@ -12,16 +13,24 @@ export { UxComponent } from './components/ux-component';
 
 export { UxTheme } from './styles/ux-theme';
 export { StyleEngine } from './styles/style-engine';
+export { GlobalStyle } from './styles/global-style';
+export { GlobalStyleEngine } from './styles/global-style-engine';
 
 export { AureliaUX } from './aurelia-ux';
+export { UXConfiguration } from './ux-configuration';
 
 export function configure(config: FrameworkConfiguration, callback?: (config: AureliaUX) => Promise<any>) {
   const ux = config.container.get(AureliaUX) as AureliaUX;
 
+  config.globalResources([
+    PLATFORM.moduleName('./reset.css')
+  ]);
+
```

Comment:
I'm thinking that you want to use `PLATFORM.moduleName` for Webpack inspection here, but you don't actually want to pass it to the `globalResources` function, since that will cause it to be loaded no matter what. I think you really want that to be controlled by the configuration below.

Reply:
So one question I have is this, without that line there the CSS file is not actually included with the code (at least in webpack). How can I ensure that the file is included when the app is built?

Reply:
I think you can just put `PLATFORM.moduleName('./reset.css')` in the code by itself and the webpack plugin will detect it and bundle.